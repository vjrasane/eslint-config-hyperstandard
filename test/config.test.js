import config from '../src/.hyperstandard.eslintrc.js'

it('validate config', () => {
  expect(config).toMatchSnapshot()
})
