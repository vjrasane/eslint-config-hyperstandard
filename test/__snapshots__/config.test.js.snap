// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate config 1`] = `
Object {
  "env": Object {
    "es6": true,
    "node": true,
  },
  "extends": "standard",
  "parser": "babel-eslint",
  "rules": Object {
    "import/prefer-default-export": Array [
      "error",
    ],
    "max-len": Array [
      "error",
      Object {
        "code": 150,
      },
    ],
    "no-await-in-loop": "error",
    "no-console": "warn",
    "no-underscore-dangle": Array [
      "error",
    ],
    "prefer-arrow-callback": "error",
    "prefer-const": "error",
    "prefer-destructuring": "error",
    "prefer-template": "error",
    "require-await": "error",
  },
}
`;
